### ISSUE TITLE
Project data language selector UI can be improved

### ISSUE THREAD

**Issue by seav** [Cadasta/cadasta-platform#1282]
1. When a user has no permission to view project data, the project data language selector still appears even though it's useless for the user.
2. On smaller screens, the selector and its tool tip are not displayed properly.

![screen shot 2017-03-11 at 00 52 35](https://cloud.githubusercontent.com/assets/873653/23804584/77f27d52-05f5-11e7-97f9-19904debb9c5.png)

**Comment by sourabhtk37** [comment]
Taking this up

**Comment by oliverroick** [comment]
@sourabhtk37 Please provide an outline how you want to approach to fix this issue before starting the work. Please refer to the [Issue Contribution Process](https://github.com/Cadasta/cadasta-project-ideas/wiki/GSoC-Outreachy-Issue-Contribution-Process) for more information

**Comment by sourabhtk37** [comment]
Simple:
Put a check on template checking whether the user has permission or not.
something like {% if user.permissions % }.
Please do suggest any other method. @oliverroick

**Comment by oliverroick** [comment]
@sourabhtk37 How will you find out how a user has permission or not?

**Comment by sourabhtk37** [comment]
tutelary can check for the permission I suppose. Can you guide me as the code is not well commented(I can help you in commenting the code, but I need a better understanding from something like a sitemap).

**Comment by kera99** [comment]
Hi @oliverroick @sourabhtk37 @seav @wonderchook ,
please is there any update on this? 
I would like to work on this, if there are no updates.

Thanks

**Comment by sourabhtk37** [comment]
@kera99 This is a low priority bug, so focus is on other issues rt now but feel free to share your approach.

**Comment by kera99** [comment]
@sourabhtk37 Here is how I would go about the issue:

- In the organization/views/default.py, I will make sure that the require permission is passed in the context data of the page displayed. If it is not, I shall add it(The permissions are stored in the db).

- In the template concerned, I shall isolate the header block that contains the language selection. I shall basically put it in a seperated file and include it in the block of the parent page. Then, in the child page, I shall override this block by passing the page name as a variable in the header block.

This way, I shall have a seperate header file that can be configured and overriden from the child pages.

**Comment by clash99** [comment]
Making adjustments to css file for smaller screens.

**Comment by oliverroick** [comment]
We need to verify if this is still an issue. I believe both bugs should be fixed now.

**Comment by clash99** [comment]
Also related is that inlinemanual launcher sits on top open user menu.

<img width="141" alt="screenshot 2017-09-21 16 23 01" src="https://user-images.githubusercontent.com/1119621/30719480-5747930c-9ee9-11e7-9c8d-3a6317258314.png">

**Comment by clash99** [comment]
Updated z-index for launcher in inlinemanual on all three sites.

**Comment by clash99** [comment]
CSS is fixed for responsive: 

<img width="670" alt="screenshot 2017-09-21 16 39 24" src="https://user-images.githubusercontent.com/1119621/30720031-930758f8-9eeb-11e7-9f33-e5551aff13e3.png">
<img width="374" alt="screenshot 2017-09-21 16 39 17" src="https://user-images.githubusercontent.com/1119621/30720030-9304d39e-9eeb-11e7-9286-90ef12250ee4.png">
<img width="842" alt="screenshot 2017-09-21 16 40 43" src="https://user-images.githubusercontent.com/1119621/30720060-a58f0264-9eeb-11e7-9a36-0b64dc7c6e3a.png">
